pretend/src/resolver.rs

```diff
@@ -24,6 +24,7 @@ pub trait ResolveUrl {
 /// Default URL resolver
 ///
 /// This resolver appends the path to a base URL.
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Same as for Pretend, this is a rather technical struct, so Clone, Debug make sense, but not really Eq, Ord and Hash.